scripts/rollup/build.js

```diff
@@ -194,13 +194,21 @@ function getFilename(name, hasteName, bundleType) {
   }
 }
 
-function uglifyConfig({
-  mangle,
-  manglePropertiesOnProd,
-  preserveVersionHeader,
-  removeComments,
-  headerSanityCheck,
-}) {
+/**
+ * @param configs {
```

Comment:
I'm also curious about the motivation — is it because the code is hard to read? Or it doesn't run in some Node version? It it's about the declaration feeling crowded, maybe we could keep the destructuring but move it into a first statement. I'm just hesitant to add comments that aren't checked by a tool.